style.css

```diff
@@ -0,0 +1,108 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Poppins', sans-serif;
+}
+
+.nav_bar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  margin-left: 10%;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+.nav_bar ul {
+  display: flex;
+  list-style: none;
+  gap: 30px;
+  margin-right: 10%;
+  font-size: 15px;
+  font-family: serif;
+  font-weight: 500;
+}
+
+.nav_bar li a {
+  text-decoration: none;
+  color: #344563;
+  font-family: 'Poppins', sans-serif;
+}
+
+.burger-icon {
+  display: none;
+  margin-right: 10%;
+}
+
+@media (max-width: 768px) {
+  .burger-icon {
+    display: flex;
+  }
+
+  .nav_bar ul {
+    display: none;
+  }
+}
+
+@media (max-width: 375px) {
+  .burger-icon {
+    display: flex;
+  }
+
+  .nav_bar ul {
+    display: none;
+  }
+}
+
+.section {
+  background: url("img/Header bg@2x.png") no-repeat;
+  background-size: contain;
```

Comment:
- [ ] Excellent job applying styles to your headline section 🚀. However, please kindly make sure that you stick as much as possible to the Figma design provided 🤓. Currently, I notice 2 main differences between your design and the Figma design:
    - Your background has multiple geometric patterns in a different location (red arrows).
    - Your title and paragraph have different font sizes and font types (green rectangles).

This is your design 👇🏽

![design](https://user-images.githubusercontent.com/116701485/235040344-f96fcd88-8b7e-4c43-9ef4-938eff6b08ac.png)

This is the Figma design 👇🏽 (almost there 😅)

![figma](https://user-images.githubusercontent.com/116701485/235040332-1f54309f-5098-4f35-905d-cfcf98bfdb31.png)